### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by cc4git** [xstefank/gmail-row-highlight#11]
If I choose a message from the Inbox, and then return to the Inbox via the "Back to Inbox" arrow, there's no indicator telling me which message I just read. however, activating "keyboard shortcuts" will cause a Blue line indicator to appear next to the last message you read. I really like this, except I wish it stood out more. Is there a way to highlight the entire row of the last message you read...

**Comment by xstefank** [comment]
Definitely, this should be possible, I will try to get to it if time permits
